azurerm/resource_arm_function_app.go

```diff
@@ -224,6 +224,27 @@ func resourceArmFunctionApp() *schema.Resource {
 					},
 				},
 			},
+
+			"functions": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"key": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
```

Comment:
Could we mark this as sensitive?

Reply:
Good call, moved the structure but have added this to the new data types schema.